_build/html/genindex.html

```diff
@@ -355,7 +355,7 @@ <h1 id="index">Index</h1>
   <div class="footer-item">
     
 <p class="component-author">
-By The Bernal Research Group
+By SECQUOIA
```

Comment:
Considering the Acronym of SECQUOIA, which is System Engineering via Classical and Quantum Optimization for Industrial Application. It would be better to keep the Research Group.

I am a little bit hesitant about adding the hyperlink of SECQUOIA. 
![image](https://github.com/user-attachments/assets/cc99663e-491a-4cb0-92d5-b2037926b341)

From the image, I think the main function is to let others know about the copyright.